File: 22/save-money.js
function finalAmount(capital, time, rate) {
  return capital * Math.pow(1 + rate, time)
}

const capital = parseFloat(prompt('How much are you going to save this month?'))
const time = parseInt(prompt('For how many months?'))
const interest = parseFloat(prompt('How much is the monthly interest (0 - 100%)')) / 100

alert(`In ${time} month(s) you gonna save $${finalAmount(capital, time, interest).toFixed(2)}`)

File: 22-day/save-money.py
def finalAmount(capital, time, rate):
    return capital * (1 + rate) ** time 


capital = float(input('How much are you going to save this month: '))
time = float(input('For how many months: '))
interest = float(input('How much is the monthly interest (0 - 100%) '))/100

print('In {} month(s) gonna save ${}'.format(time, round(finalAmount(capital, time, interest), 2)))